### PR TITLE
Analytics cron

### PR DESCRIPTION
**Pull request by nathan-moore-97**

### CODE REVIEW

File: .github/workflows/deploy-cron.yaml [hunk 1]
```diff
@@ -31,11 +31,14 @@ jobs:
       - name: Install Cron
         run: ./install_cron.sh
         working-directory: ./cron
+      
+      - name: Set enviornment name
+        run: cf push --var ENVIRONMENT=${{ steps.cf-setup.outputs.target-environment }} --strategy rolling
```
Comment (dzaslavskiy) [inline .github/workflows/deploy-cron.yaml:36]: This step is a duplicate of the one below

File: cron/manifest.yaml [hunk 2]
```diff
@@ -13,5 +13,5 @@ applications:
       - cron-service-account
       - outbound-proxy
     env:
-      GDRIVE_APP_HOST: "idva-gdrive-((ENVIRONMENT_NAME)).apps.internal"
+      GDRIVE_APP_HOST: "idva-gdrive-((ENVIRONMENT)).apps.internal"
```
Comment (dzaslavskiy) [inline cron/manifest.yaml:16]: This change has already been commited

**Review by dzaslavskiy — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy-cron.yaml (hunk 1 above)
Comment (nathan-moore-97) [reply .github/workflows/deploy-cron.yaml:36]: I dont understand what you mean here

File: .github/workflows/deploy-cron.yaml [hunk 3]
```diff
@@ -31,11 +31,14 @@ jobs:
...
```
Comment (nathan-moore-97) [inline .github/workflows/deploy-cron.yaml:37]: Missing working directory

File: .github/workflows/deploy-cron.yaml (hunk 1 above)
Comment (nathan-moore-97) [reply .github/workflows/deploy-cron.yaml:36]: Oh, I see

File: cron/manifest.yaml (hunk 2 above)
Comment (nathan-moore-97) [reply cron/manifest.yaml:16]: I dont know why its here again, and not sure what I should do to fix it

Comment (dzaslavskiy) [reply cron/manifest.yaml:16]: same as we did before

**Review by dzaslavskiy — APPROVED** [review]

**Pull request merged**